src/components/upload/nz-upload-list-item-icon-and-preview.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, Input, Output, EventEmitter, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'nz-upload-list-item-icon-and-preview',
+  templateUrl: './nz-upload-list-item-icon-and-preview.component.html',
+  styleUrls: [
+    './style/index.less'
+  ],
+  encapsulation: ViewEncapsulation.None,
+})
+export class NzUploadListItemIconAndPreviewComponent implements OnInit {
+
+  _locale = {
+    uploading: '文件上传中',
+    removeFile: '删除文件',
+    uploadError: '上传错误',
+    previewFile: '预览文件',
+  };
```

Comment:
You may achieve i18n now by using the `locale` module

Reply:
Thanks,I will delete them.

Reply:
- [x] Achieve i18n by using the locale module